### DISCUSSION TITLE
dynamic field for event maximum tickets

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3120] [Q&A]
Hello

I have created some events and I would like to display the widget of these events with a parameter of maximum tickets one can buy.
This parameter should be dynamic and depends on a user's maximum tickets that he/she can buy as it is configured in a third system (other than pretix). So this cannot be done with the maximum items per order that pretix offers.

For example let's say that Alex can buy 10 tickets from a specific event in pretix. This is defined in a third CRM system.
I want to display the event widget and do not allow Alex to buy more than 10 tickets. So let's say that he buys 2 tickets. The next time he should be able to buy up to 2 tickets.

Is this possible ?

Thank you in advance.

**Comment by raphaelm** [comment]
This is not directly possible, but you could emulate it with per-user vouchers, I guess.

**Comment by alemao8** [comment]
How is a per-user voucher implemented ? Combining membership and vouchers or do u mean another way ?
Thank you in advance

**Reply by raphaelm** [reply]
I meant: Just create X vouchers and sent every person one voucher.
